guvnor-structure/guvnor-structure-api/src/main/java/org/guvnor/structure/repositories/GitMetadataStore.java

```diff
@@ -37,6 +37,14 @@ void write( String name,
                 String origin );
 
     /**
+     * Writes the meta information about a repository.
+     * @param name The name of the repository in organizationalUnit/repositoryName format
```

Comment:
as far as I understand repo name is flexible, doesn't require "ou/name" as it's possible to have repositories with just "name" or "username/reponame" too.